schemas/fluidity_options.rnc

```diff
@@ -5118,7 +5118,21 @@ scalar_field_choice =
                   }
                }
             )
-         }    
+         }|
+         element scalar_field {
```

Comment:
This needs some user documentation. What does it do, what other options does it require?